src/layers/fp64/choropleth-layer/choropleth-layer.js

```diff
@@ -75,7 +75,6 @@ export default class ChoroplethLayer64 extends Layer {
     const IndexType = gl.getExtension('OES_element_index_uint') ?
       Uint32Array : Uint16Array;
 
-    this.setUniforms({opacity: this.props.opacity});
```

Comment:
Same... opacity is a base layer attribute, and the corresponding uniform is set in base layer